src/ConnectedMode.UnitTests/Suppressions/SuppressionIssueStoreUpdaterTests.cs

```diff
@@ -169,6 +169,7 @@ public void UpdateAll_OperationCancelledException_CancellationMessageLogged()
         }
 
         [TestMethod]
+        [Ignore] // Flaky. See https://github.com/SonarSource/sonarlint-visualstudio/issues/4307
```

Comment:
I tried increasing the timeout to see if that made the test less flaky... then I read the ticket I created about the task earlier this week (#4307). It seems from the test log that the required string is present, but the test is still failing -> it requires more investigation -> disable the test for now.